### ISSUE TITLE
Split requirements.txt

### ISSUE THREAD

**Issue by owlen** [TheCheapestPixels/wecs#15]
Following an offline discussion with @Schwarzbaer :
Since the WECS can work without panda, and examples "minimal" and "rpg" actually demo it, I would argue that splitting the requirements.txt to "pure wecs" and "other(panda) stuff" makes sense.

I suggest a separate `wecs_req.txt` and `wecs_panda_req.txt`.

This would allow users who don't need panda to not be encumbered by it. At the same time, tests for the framework should be able to run well without the extra requirements.
As for the documentation - I would write in the regular instructions to install both files, but add a note that for non Panda usage, that extra req file is not needed.

**Comment by Schwarzbaer** [comment]
Agreed, but I'm also pondering just splitting everything Panda3D-related into its own project; `panda3d-wecs` or so. Opinions, @janEntikan ?
